libshortfin/CMakeLists.txt

```diff
@@ -126,37 +126,38 @@ endif()
 
 ## iree runtime
 
-if (NOT SHORTFIN_IREE_SOURCE_DIR AND SHORTFIN_BUNDLE_DEPS)
+# Set IREE build flags
+set(IREE_BUILD_COMPILER OFF)
```

Comment:
You could guard this by adding `if (SHORTFIN_IREE_SOURCE_DIR OR SHORTFIN_BUNDLE_DEPS)` but this isn't strictly necessary,

Reply:
done